src/testRunner/tests.ts

```diff
@@ -188,3 +188,4 @@ import "./unittests/tsserver/typingsInstaller";
 import "./unittests/tsserver/versionCache";
 import "./unittests/tsserver/watchEnvironment";
 import "./unittests/debugDeprecation";
+import "./unittests/tsserver/inconsistentErrorInEditor";
```

Comment:
Just note it, I like this filename choice... because we have more bugs like this (#49624 maybe?) which just don't seem to be reproducible in fourslash. Seems like a good bucket to throw these "tsserver is misbehaving so here's literally the trace" tests into when needed.